onyo/lib/commands.py

```diff
@@ -400,6 +403,10 @@ def onyo_get(inventory: Inventory,
         err_str = '\n'.join([str(x) for x in invalid_paths])
         raise ValueError(f"The following paths are not part of the inventory:\n{err_str}")
 
+    allowed_sorting = ['ascending', 'descending']
+    if sort not in allowed_sorting:
+        raise ValueError(f"Allowed sorting modes: {', '.join(allowed_sorting)}")
+
```

Comment:
Seeing this, reminds me: We should probably include that kind of thing in type annotations:

So, `sort: Literal['ascending', 'descending']` instead of `sort: str`.